src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -1133,21 +1134,37 @@ private void addCommentTags(Element element, List<? extends DocTree> tags, boole
         }
     }
 
-    boolean ignoreNonInlineTag(DocTree dtree) {
+    // helper methods because jdk21 functionality is not allowed
+    Name getLastHelper(List<Name> l) {
```

Comment:
This could be made static and private.

Reply:
Agree,  'private static' this is better .